src/main/java/net/minestom/server/thread/Acquirable.java

```diff
@@ -20,15 +21,12 @@ public sealed interface Acquirable<T> permits AcquirableImpl {
      *
      * @return the entities ticked in the current thread
      */
-    static @NotNull Stream<@NotNull Entity> currentEntities() {
-        final Thread currentThread = Thread.currentThread();
-        if (currentThread instanceof TickThread) {
-            return ((TickThread) currentThread).entries().stream()
-                    .flatMap(partitionEntry -> partitionEntry.elements().stream())
-                    .filter(tickable -> tickable instanceof Entity)
-                    .map(tickable -> (Entity) tickable);
-        }
-        return Stream.empty();
+    static @NotNull Stream<@NotNull Entity> locals() {
```

Comment:
should keep `localEntities` I guess because acquirable is technically not restricted to entity